src/tool/cpp/main.cpp

```diff
@@ -31,7 +32,9 @@ namespace xlang
             { "include", 0 },
             { "exclude", 0 },
             { "root", 0, 1 },
-            { "base", 0, 0 }
+            { "base", 0, 0 },
+            { "prefix", 0, 0 },
+            { "overwrite", 0, 0 }
         };
```

Comment:
these are redundant

Reply:
What's redundant?

Reply:
prefix and overwrite are already listed on lines 28, 29

---
In reply to: [229060880](https://github.com/Microsoft/xlang/pull/60#discussion_r229060880) [](ancestors = 229060880)

Reply:
Ah. I'll add a debug assertion checking for dupes.